src/main/java/employee/service/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package employee.service;
+
+import employee.data.EmployeeJPA;
+import employee.data.EmployeeMongo;
+import employee.repository.EmployeeJPARepository;
+import employee.repository.EmployeeMongoRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service("employeeService")
+public class EmployeeServiceImpl implements EmployeeService {
+
+    @Autowired
+    EmployeeJPARepository employeeJPARepository;
+
+    @Autowired
+    EmployeeMongoRepository employeeMongoRepository;
+
+    public Object getEmployee(String id, int version) {
```

Comment:
IntelliJ Should have flagged this to add `@Override` please add it because you are overriding methods from the interface

Reply:
done